google/cloud/spanner/samples/samples.cc

```diff
@@ -3414,6 +3451,18 @@ void RunAllSlowInstanceTests(
 
   std::cout << "\nRunning delete-instance sample" << std::endl;
   DeleteInstance(instance_admin_client, project_id, crud_instance_id);
+
+  if (!emulator) {
+    std::cout
+        << "\nRunning spanner_create_instance_with_processing_units sample"
+        << std::endl;
+    CreateInstanceWithProcessingUnits(
+        instance_admin_client, project_id, crud_instance_id,
+        "Test Low-Cost Instance", "regional-us-central1");
```

Comment:
I assume the hard-coded config is necessary.  Consider a TODO to pick a location at random, or this will create contention across the builds.

Reply:
Yes ... my understanding is that this will currently only work in that region.  We already have the randomly chosen `config` to change this to before merging, although the "TODO" for that is currently only in my head.  I'll make a pending edit to do that, but will deliver the existing tested version first.  Thanks.